upgrading/topics/install_new_version.adoc

```diff
@@ -2,156 +2,14 @@
-
-. To activate any new features in the new release, such as new subsystems, manually merge each .rpmnew file into your existing configuration files.
-
-. Copy any custom modules that have been added to the modules directory.
-
-. Continue with the section, xref:upgrade-script[Running the server upgrade script].
-+
-[NOTE]
-====
-{project_name} RPM server distribution is using
-
-`{project_dirref}=/opt/rh/rh-sso7/root/usr/share/keycloak`
-
-Use it when calling migration scripts below.
-====
-
-endif::[]
-
-[id="upgrade-script"]
-== Running the server upgrade script
-
-Based on your previous installation, run the appropriate upgrade script that applies to your situation:
-
-* xref:standalone-mode[Standalone mode]
-* xref:standalone-ha[Standalone high availability mode]
-* xref:domain-mode[Domain mode]
-* xref:domain-clustered[Domain-clustered mode]
-
-[id="standalone-mode"]
-=== Running the Standalone Mode upgrade script
-
-.Procedure
-
-. If you are using a different configuration file than the default one, edit the migration script to specify the new file name.
-. Stop the server.
-. Run the upgrade script:
-
- bin/jboss-cli.sh --file=bin/migrate-standalone.cli
-
-[id="standalone-ha"]
-=== Running the Standalone-High Availability Mode upgrade script
-For standalone-high availability (HA) mode, all instances must be upgraded at the same time.
-
-.Procedure
-. If you are using a different configuration file than the default one, edit the migration script to specify the new file name.
-. Stop the server.
-. Run the upgrade script:
-
- bin/jboss-cli.sh --file=bin/migrate-standalone-ha.cli
-
-[id="domain-mode"]
-=== Running the Domain Mode upgrade script
-For domain mode, all instances must be upgraded at the same time.
-
-.Procedure
-
-. If you have changed the profile name, you must edit the upgrade script to change a variable near the beginning of the script.
-. Edit the domain script to include the location of the keycloak-server.json file.
-. Stop the server.
-. Run the upgrade script on the domain controller
-
- bin/jboss-cli.sh --file=bin/migrate-domain.cli
-
-[id="domain-clustered"]
-=== Running the Domain-clustered Mode upgrade script
-For domain-clustered mode, all instances must be upgraded at the same time.
-
-.Procedure
-
-. If you have changed the profile name, you must edit the upgrade script to change a variable near the beginning of the script.
-. Edit the domain-clustered script to include the location of the keycloak-server.json file.
-. Stop the server.
-. Run the upgrade script on the domain controller only:
-
- bin/jboss-cli.sh --file=bin/migrate-domain-clustered.cli
+. Copy `conf/`, `providers/` and `themes/` from the previous installation to the new installation.
+. Re-build the server with `bin/kc.[sh|bat] build` unless you are using auto-build
```

Comment:
```suggestion
```

Reply:
As you know, we don't have auto build anymore and we can just remove that line I think. In a fresh distribution (updated from a previous distribution) it should be just a matter of starting the server to have everything set.